test/WebJobs.Script.Tests/Workers/Rpc/RpcFunctionInvocationDispatcherTests.cs

```diff
@@ -318,13 +318,28 @@ public async Task FunctionDispatcher_DoNot_Restart_ErroredChannels_If_WorkerRunt
         [InlineData(@"node", true, true, true)]
         [InlineData(@"node", false, false, false)]
         [InlineData(@"java", false, true, false)]
+        [InlineData("", false, true, false)]
         public async Task FunctionDispatcher_ShouldRestartChannel_Returns_True(string language, bool isWebHostChannel, bool isJobHostChannel, bool expectedResult)
         {
             RpcFunctionInvocationDispatcher functionDispatcher = GetTestFunctionDispatcher();
             await functionDispatcher.InitializeAsync(GetTestFunctionsList(RpcWorkerConstants.NodeLanguageWorkerName));
             Assert.Equal(expectedResult, functionDispatcher.ShouldRestartWorkerChannel(language, isWebHostChannel, isJobHostChannel));
         }
 
+        [Theory]
+        [InlineData(@"node", false, true, false)]
+        [InlineData(@"node", true, false, false)]
+        [InlineData(@"node", true, true, false)]
+        [InlineData(@"node", false, false, false)]
+        [InlineData(@"java", false, true, false)]
+        [InlineData("", false, true, false)]
+        public async Task FunctionDispatcher_ShouldNotRestartChannel_InPlaceholderMode(string language, bool isWebHostChannel, bool isJobHostChannel, bool expectedResult)
+        {
+            RpcFunctionInvocationDispatcher functionDispatcher = GetTestFunctionDispatcher(isPlaceholderMode: true);
+            await functionDispatcher.InitializeAsync(null);
+            Assert.Equal(expectedResult, functionDispatcher.ShouldRestartWorkerChannel(language, isWebHostChannel, isJobHostChannel));
```

Comment:
instead of adding this test, update earlier test to either initilize function dispatcher with a language or null

Reply:
@pragnagopa - done!